components/camel-fhir/camel-fhir-api/pom.xml

```diff
@@ -40,6 +40,11 @@
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-support</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-text</artifactId>
+            <version>${commons-text-version}</version>
+        </dependency>
```

Comment:
Even if it is not really necessary, I would tend to exclude it from `hapi-fhir-base` to make it clear that it is an override or at least add a comment to explain that it is an override because otherwise over time we could remove it by mistake since it is not directly called by the component. For the component, it is a runtime/transitive dependency.

Reply:
Latest commit adds an exclusion + comment.